drvs/discord-plugged.nix

```diff
@@ -1,5 +1,6 @@
 { symlinkJoin
 , discord-canary
+, discord ? discord-canary
```

Comment:
Going to adjust this name when I merge.

Works fine because you're explicitly setting it above but gives the wrong impression as if you did use this with callPackage without providing it it'll default to stable discord and the canary arg above never gets used.